helm/sematic-server/templates/configmap.yaml

```diff
@@ -21,6 +21,12 @@ data:
   AWS_S3_REGION: {{ .Values.aws.bucket_region | quote }}
   STORAGE: "sematic.plugins.storage.s3_storage.S3Storage"
 {{ end }}
+{{ if .Values.local_storage.enabled }}
+  STORAGE: "sematic.plugins.storage.local_storage.LocalStorage"
+  {{ if .Values.local_storage.local_storage_path }}
+    LOCAL_STORAGE_PATH: {{ .Values.local_storage.local_storage_path | quote }}
+  {{ end }}
```

Comment:
Why is this indented?

Reply:
Oh, didn't realize there was no indentation in this templating language. Fixing.